libs/wallet/src/web3-react/hooks/useSwitchNetwork.ts

```diff
@@ -6,12 +6,14 @@ import { useWeb3React } from '@web3-react/core'
 import { switchChain } from '../utils/switchChain'
 
 export function useSwitchNetwork() {
-  const { connector } = useWeb3React()
+  const { connector, account } = useWeb3React()
 
   return useCallback(
-    (targetChain: SupportedChainId) => {
+    async (targetChain: SupportedChainId) => {
```

Comment:
Why is this async now?
I don't see any changes requiring it to be.

Reply:
Yes, this is not super noticable :) 
As you can see, we do `if (!account) return` which returns `undefined`.
After that we return `switchChain()` with is a `Promise`.
Adding `async` makes the callback returning `Promise` in both cases.